src/Compilers/CSharp/Portable/Binder/Binder.cs

```diff
@@ -168,6 +168,18 @@ internal virtual ImmutableArray<LocalFunctionSymbol> GetDeclaredLocalFunctionsFo
             return this.Next.GetDeclaredLocalFunctionsForScope(scopeDesignator);
         }
 
+        /// <summary>
+        /// If this binder owns a scope for locals, return syntax node that is used
+        /// as the scope designator. Otherwise, null.
+        /// </summary>
+        internal virtual SyntaxNode ScopeDesignator
```

Comment:
I am not sure what are you referring to. This is a concept that binder uses internally in a couple of related APIs.